pkg/instrumentation/helper.go

```diff
@@ -14,7 +14,30 @@
 
 package instrumentation
 
-import corev1 "k8s.io/api/core/v1"
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+func defaultInitContainerResourceRequirements() corev1.ResourceRequirements {
+	const (
```

Comment:
it was intended to keep the values together in one place. But since they are  used exactly once, I have now removed them and inlined them instead.

I don't think it makes sense to provide the constants in the package scope, since they are not needed anywhere else.